### PR TITLE
Refs #27079 -- Used addClassCleanup() in SeleniumTestCase.

### PR DESCRIPTION
**Pull request by sarahboyce**
(will need to experiment as to what the issue is, will update the PR when it is ready)

### CODE REVIEW

File: .github/workflows/selenium.yml [hunk 1]
```diff
@@ -70,4 +70,4 @@ jobs:
       - name: Run Selenium tests
         working-directory: ./tests/
         run: |
-          python -Wall runtests.py --verbosity 2 --noinput --selenium=chrome --headless --settings=test_postgres --parallel 2
```
Comment (felixxm) [inline .github/workflows/selenium.yml:73]: It will run in parallel by default, so we need to pass `--parallel 1` to avoid this.

Comment (felixxm) [reply .github/workflows/selenium.yml:73]: Also, thread errors are only on SQLite.

**Comment by sarahboyce** [comment]
Re-ran a couple of times, I think the thread error is gone (you can re-run by removing the label "selenium" and adding it back a few seconds later) :crossed_fingers:

**Comment by felixxm** [comment]
@sarahboyce Thanks :+1: 

This is probably a regression in def09bf4126d4886413adf7388882eca8e32576b.

**Pull request merged**